### PR TITLE
feat(client): add splitpanes for resizable columns

### PR DESCRIPTION
**Pull request by huang-julien**
Hi :wave: ! Following https://github.com/nuxt/devtools/pull/82 , this PR adds splitpanes to the client for resizable columns.

### CODE REVIEW

File: packages/devtools/client/layouts/default.vue [hunk 1]
```diff
@@ -12,11 +14,13 @@ const waiting = computed(() => !client.value && !showConnectionWarning.value)
...
+    <Splitpanes v-else h-full of-hidden>
+      <Pane class="default-theme" min-size="5" size="20" of-auto>
+        <Drawer />
+      </Pane>
+      <Pane h-full of-auto min-size="5">
         <slot />
-      </div>
-    </div>
+      </Pane>
+    </splitpanes>
```
Comment (manniL) [inline packages/devtools/client/layouts/default.vue:24]: nit
```suggestion
    </Splitpanes>
```

**Review by manniL — APPROVED** [review]

File: packages/devtools/client/layouts/default.vue (hunk 1 above)
Comment (huang-julien) [reply packages/devtools/client/layouts/default.vue:24]: oops, thanks !

**Comment by antfu** [comment]
I feel that the resize on the main layout is not good enough. We should let the responsiveness design react to the panel size to switch from (full tab/icon only). Maybe using container queries or a programmatic approach.

**Comment by antfu** [comment]
Reverted it for now, so we can move forward and improve later. Thanks a lot for the work.

A few TODOs I see:

- Responsive main NavPanel
- Persistent panel size (local storage + maybe a general component with key)

**Pull request merged**